README.md

```diff
@@ -175,6 +175,89 @@ We are using the [Arch User Repository](https://aur.archlinux.org) to distribute
 
 You can use [pacman](https://www.archlinux.org/pacman/) to install the packages.
 
+#### RHEL/CentOS
+
+There are a few subtle yet important things to getting this setup, at the time of this writing the package cannot be installed via yum (its a feature request). So this is how we setup this on RHEL following some best practices.
+
+1. [Required] Install `step`.
+
+    Download the latest Linux package from
+    [`step` releases](https://github.com/smallstep/cli/releases):
+
+    ```
+    $ wget -O step-cert.tar.gz https://github.com/smallstep/cli/releases/download/vX.Y.Z/step_linux_X.Y.Z_amd64.tar.gz
```

Comment:
We use `step-cert` here, and the `step` and `step-` in the code block in the following step. Is that correct?